### PR TITLE
Re-word philosophy

### PR DESCRIPTION
**Pull request by arctangent**
"Iterative and incremental" is the usual way of describing the Agile approach. The rapidity of the approach is a second-order effect.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The framework is concerned with using "good" software engineering to support rap
 
 The philosophy underpinning this framework advocates:
 * Empowered teams which are accountable for their products: "You build it, you run it" (Werner Vogels)
-* Rapid and iterative development: "Release early. Release often. And listen to your customers." (Eric S. Raymond in [The Cathedral and the Bazaar](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar))
+* Iterative and incremental development: "Release early. Release often. And listen to your customers." (Eric S. Raymond in [The Cathedral and the Bazaar](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar))
```
Comment (ivorc) [inline README.md:15]: I don't disagree with the thinking, but I personally think the original wording captures the flavour of what we want to say better.

**Review by andyblundell — COMMENTED** [review]
Hi - thanks for the feedback. I'm happy to add "incremental" but would like to keep "rapid" as we're talking about more than just being agile: moving at pace is at the heart of the philosophy here.

"Rapid, iterative and incremental development"?

**Comment by arctangent** [comment]
Works for me :-)

**Comment by andyblundell** [comment]
New wording here:

https://github.com/NHSDigital/software-engineering-quality-framework/pull/30

... not sure if the intention was to update this PR with the new wording

**Comment by andyblundell** [comment]
Closing as Ivor has merged PR30

**Pull request closed without merging**